### PR TITLE
BOM-2989: Replace Travis with GitHub CI

### PR DESCRIPTION
**Pull request by UsamaSadiq**
**Issue:** [BOM-2989](https://openedx.atlassian.net/browse/BOM-2989)

### Description
- Replace `Travis` with `GitHub CI`.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        python-version:
+          - 3.8
+        toxenv: [ testenv ]
+    steps:
+    - uses: actions/checkout@v1
+    - name: setup python
```
Comment (nedbat) [inline .github/workflows/ci.yml:24]:
```suggestion
    - name: Set up Python
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        python-version:
+          - 3.8
+        toxenv: [ testenv ]
+    steps:
+    - uses: actions/checkout@v1
```
Comment (nedbat) [inline .github/workflows/ci.yml:23]: I think it's nicer if every step has a name.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        python-version:
+          - 3.8
+        toxenv: [ testenv ]
```
Comment (nedbat) [inline .github/workflows/ci.yml:21]: Why does this field use `[]` syntax while the others use `-` syntax?  It will be easier to read if they are the same.

**Review by nedbat — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (UsamaSadiq) [reply .github/workflows/ci.yml:19]: Updated all the fields to use `[]` notation.

**Review by nedbat — APPROVED** [review]

**Pull request merged**